src/main/java/com/cosmeticos/service/OrderService.java

```diff
@@ -174,10 +174,15 @@ private void addInWallet(Professional professional, Customer customer) {
 		}
 	}
 
-	public Order update(OrderRequestBody request) {
+	public Order update(OrderRequestBody request) throws Exception {
 		Order orderRequest = request.getOrder();
 		Order order = orderRepository.findOne(orderRequest.getIdOrder());
 
+		//ADICIONEI ESSA VALIDACAO DE TENTATIVA DE ATUALIZACAO DE STATUS PARA O MESMO QUE JA ESTA EM ORDER
+		if(order.getStatus() == orderRequest.getStatus()) {
+			throw new IllegalStateException("PROIBIDO ATUALIZAR PARA O MESMO STATUS.");
```

Comment:
Se for pra lançar OrderValidationException, então não seria melhor já tratar isso dentro de Validation?

E mais, se lançar isso, terá que tratar qual a causa da exceção e responder corretamente, caso contrário, o responseBody para o cliente está definido manualmente como abaixo:

orderResponseBody.setDescription("Erro: Profissional não pode ter duas Orders simultaneas em andamento. - {}: " + errorCode);